docarray/array/mixins/traverse.py

```diff
@@ -42,7 +105,8 @@ def traverse(
             - `r,c`: docs in this TraversableSequence and all child-documents at granularity 1
 
         """
-        for p in traversal_paths.split(','):
```

Comment:
I guess there must be some update of the `docstrings`

Reply:
updated